### PR TITLE
Fixed version number according to TUF spec

### PR DESCRIPTION
**Pull request by jonasclaes**
Fixed version number according to TUF specification as of 2 december, 2022.

### CODE REVIEW

**Review by kommendorkapten — APPROVED** [review]

File: src/models/base.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { guard } from '../utils';
 import { JSONObject, JSONValue } from '../utils/types';
 import { Signature } from './signature';
 
-const SPECIFICATION_VERSION = ['1', '20', '30'];
+const SPECIFICATION_VERSION = ['1', '0', '30'];
```
Comment (joshuagl) [inline src/models/base.ts:7]: Nit: the current version of the [specification is 1.0.31](https://theupdateframework.github.io/specification/v1.0.31/index.html), the only notable change from 1.0.30 is that we simplified how we describe the root rollback check in 5.3.5 of the detailed client workflow (see https://github.com/theupdateframework/specification/commit/7b565c71c4b6b95a303dc75f7efbfab7489e53c4).

**Review by joshuagl — APPROVED** [review]
1.0.30 isn't the latest version (see my review comment) but this seems fine to me. 

FYI we have a [reusable workflow](https://github.com/theupdateframework/specification/blob/2443fbafc77c8a805b9b727973777e97b5f24334/.github/workflows/check-latest-spec-version.yml) defined in the specification repository which can be used (i.e., from a [cron workflow as we do in python-tuf](https://github.com/theupdateframework/python-tuf/blob/2a02c8ba76e48f3ccc624c085d84f09f1adb16ab/.github/workflows/specification-version-check.yml)) to compare the current specification version in your implementation to the latest published version and file an issue against your implementation's GitHub Issues when a newer spec version is available.

**Comment by jonasclaes** [comment]
> 1.0.30 isn't the latest version (see my review comment) but this seems fine to me.
> 
> FYI we have a [reusable workflow](https://github.com/theupdateframework/specification/blob/2443fbafc77c8a805b9b727973777e97b5f24334/.github/workflows/check-latest-spec-version.yml) defined in the specification repository which can be used (i.e., from a [cron workflow as we do in python-tuf](https://github.com/theupdateframework/python-tuf/blob/2a02c8ba76e48f3ccc624c085d84f09f1adb16ab/.github/workflows/specification-version-check.yml)) to compare the current specification version in your implementation to the latest published version and file an issue against your implementation's GitHub Issues when a newer spec version is available.

Thanks! I didn't even see that, that's usefull! I've updated the version, it should be correct now.

**Review by joshuagl — APPROVED** [review]
:+1:

**Comment by jonasclaes** [comment]
Can you approve running the workflows so that I can merge this?

**Pull request merged**